runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -82,13 +82,17 @@
 	);
 
 	private static final Set<Integer> BARROWS_LADDERS = Sets.newHashSet(NullObjectID.NULL_20675, NullObjectID.NULL_20676, NullObjectID.NULL_20677);
+	private final Set<Widget> POSSIBLE_SOLUTIONS = new HashSet<>();
```

Comment:
this should not be upper case. also, why this even exists? delete this entire set